sakiparens.py

```diff
@@ -0,0 +1,17 @@
+from __future__ import unicode_literals
+
+def parens(string):
+    stack = []
+    for char in string:
+        if char == '(':
+            stack.append(char)
+        elif char == ')':
+            try:
+                stack.pop()
+            except:
```

Comment:
You probably want to catch a specific exception.  What is raised by the stack if `.pop()` is called on an empty stack?  That's the error you should be catching, not just any error.